backend/src/main/java/org/eclipse/tractusx/puris/backend/common/security/SecurityConfig.java

```diff
@@ -96,6 +96,7 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                         "/planned-production/**",
                         "/material-demand/**",
                         "/delivery-information/**",
+                        "/supply/**",
```

Comment:
Please rename to `days-of-supply`. Things like supply status are also in discussion and likely this will be a different route